examples/sycl/pvc/flash_attention_v2/pvc_flash_attn_epilogue.hpp

```diff
@@ -260,8 +264,9 @@ class CollectiveEpilogueAttention<
     // work items within subgroup have the same sum() data stored
     // in registers
     CUTLASS_PRAGMA_UNROLL
-    for (int i = 0; i < cute::size(tLSEr); i += SubgroupSize) {
-      *(lse_ptr + i + lane_id) = tLSEr(i);
+    for (int i = 0; i < cute::size(sum); i += SubgroupSize) {
+      // Tensor tLSEr replacement by sum ;
+      *(lse_ptr + i + lane_id) = sum(i);
```

Comment:
does this produce block write?

Reply:
Thanks for catching that, this should be removed from the code as this is for the forward pass of training and not inference. It should not be added in this example, in the current shape